src/pages/settings/Profile/PersonalDetails/StateSelectionPage.tsx

```diff
@@ -65,11 +65,11 @@ function StateSelectionPage() {
                     Navigation.goBack();
                 } else {
                     // "backTo" provided: navigate back to "backTo" with state parameter.
-                    Navigation.goBack(appendParam(backTo, 'state', option.value) as Route);
```

Comment:
Same here and below with the type. Is it okay to take out or should we use a different type?

Reply:
consolidating answer [here](https://github.com/Expensify/App/pull/44112#discussion_r1649436515)